dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild

```diff
@@ -0,0 +1,287 @@
+
+IUSE="doc client superserver xinetd examples debug"
+REQUIRED_USE="
+	client? ( !superserver )
+	client? ( !xinetd )
+	superserver? ( !xinetd )
+"
+
+CDEPEND="
+	dev-libs/libedit
+	dev-libs/icu:=
+"
+DEPEND="${CDEPEND}
+	>=dev-util/btyacc-3.0-r2
+	doc? ( app-arch/unzip )
+"
+RDEPEND="${CDEPEND}
+	xinetd? ( virtual/inetd )
+	!sys-cluster/ganglia
+"
+
+RESTRICT="userpriv"
+
+S="${WORKDIR}/${MY_P}"
+
+pkg_setup() {
+	enewgroup firebird 450
+	enewuser firebird 450 /bin/bash /usr/$(get_libdir)/firebird firebird
+}
+
+function check_sed() {
+	MSG="sed of $3, required $2 lines modified $1"
+	einfo "${MSG}"
+	[[ $1 -ge $2 ]] || die "${MSG}"
+}
+
+src_unpack() {
+	unpack "${MY_P}.tar.bz2"
+	if use doc; then
+		# Unpack docs
+		mkdir "${WORKDIR}/manuals" && cd "${WORKDIR}/manuals" || die
+		unpack ib_b60_doc.zip
+	fi
+}
+
+src_prepare() {
+	# This patch might be portable, and not need to be duplicated per version
+	# also might no longer be necessary to patch deps or libs, just flags
+	epatch "${FILESDIR}"/${PN}-2.5.3.26780.0-deps-flags.patch
+
+	use client && epatch "${FILESDIR}"/${PN}-2.5.1.26351.0-client.patch
+	use superserver || epatch "${FILESDIR}"/${PN}-2.5.1.26351.0-superclassic.patch
+
+	# Rename references to isql to fbsql
+	# sed vs patch for portability and addtional location changes
+	check_sed "$(sed -i -e 's:"isql :"fbsql :w /dev/stdout' \
+		src/isql/isql.epp | wc -l)" "1" "src/isql/isql.epp" # 1 line
+	check_sed "$(sed -i -e 's:isql :fbsql :w /dev/stdout' \
+		src/msgs/history2.sql | wc -l)" "4" "src/msgs/history2.sql" # 4 lines
+	check_sed "$(sed -i -e 's:--- ISQL:--- FBSQL:w /dev/stdout' \
+		-e 's:isql :fbsql :w /dev/stdout' \
+		-e 's:ISQL :FBSQL :w /dev/stdout' \
+		src/msgs/messages2.sql | wc -l)" "6" "src/msgs/messages2.sql" # 6 lines
+
+	find "${S}" -name \*.sh -print0 | xargs -0 chmod +x || die
+	rm -rf "${S}"/extern/{btyacc,editline,icu} || die
+
+	eautoreconf
+}
+
+src_configure() {
+	filter-flags -fprefetch-loop-arrays
+	filter-mfpmath sse
+
+	econf \
+		--prefix=/usr/$(get_libdir)/firebird \
+		$(use_enable superserver superserver) \
+		$(use_enable debug) \
+		--with-editline \
+		--with-system-editline \
+		--with-system-icu \
+		--with-fbbin=/usr/bin \
+		--with-fbsbin=/usr/sbin \
+		--with-fbconf=/etc/${PN} \
+		--with-fblib=/usr/$(get_libdir) \
+		--with-fbinclude=/usr/include \
+		--with-fbdoc=/usr/share/doc/${P} \
+		--with-fbudf=/usr/$(get_libdir)/${PN}/UDF \
+		--with-fbsample=/usr/share/doc/${P}/examples \
+		--with-fbsample-db=/usr/share/doc/${P}/examples/db \
+		--with-fbhelp=/usr/$(get_libdir)/${PN}/help \
+		--with-fbintl=/usr/$(get_libdir)/${PN}/intl \
+		--with-fbmisc=/usr/share/${PN} \
+		--with-fbsecure-db=/etc/${PN} \
+		--with-fbmsg=/usr/$(get_libdir)/${PN} \
+		--with-fblog=/var/log/${PN}/ \
+		--with-fbglock=/var/run/${PN} \
+		--with-fbplugins=/usr/$(get_libdir)/${PN}/plugins \
+		--with-gnu-ld \
+		${myconf}
+}
+
+src_compile() {
+	MAKEOPTS="${MAKEOPTS/-j*/-j1} ${MAKEOPTS/-j/CPU=}"
```

Comment:
There is a reason it is done as it is, you would have to back in old cvs history to see when that was added/changed and the message. You could be right, but why not test vs guessing?